crates/yttrium/src/smart_accounts/safe.rs

```diff
@@ -393,12 +393,12 @@ pub fn prepare_sign(
 #[cfg_attr(feature = "uniffi", derive(uniffi::Enum))]
 pub enum SignOutputEnum {
     Signature(Bytes),
-    SignOutput(SignOutput),
+    SignOutput(SignOutputObject),
```

Comment:
```suggestion
    // renamed to `Object` to avoid conflicts: https://github.com/mozilla/uniffi-rs/issues/2402
    SignOutput(SignOutputObject),
```